.github/workflows/test.yml

```diff
@@ -148,6 +167,10 @@ jobs:
       run: |
         vagrant up
         vagrant ssh-config >> ~/.ssh/config
+    - name: move cosign binary to /usr/local/bin
+      run: |
+         ssh default -- "sudo mv /vagrant/cosign /usr/local/bin"
+         ssh default -- "cosign version"
```

Comment:
If you want to install cosign into the vagrant VM, this has to be done in `Vagrantfile`.

However, I think you can just skip cosign tests for Vagrant, as cosign isn't related to cgroup stuffs AFAICS.

Reply:
okay, thank you, done.